cmd/caa-log-checker/main.go

```diff
@@ -177,12 +192,28 @@ func main() {
 	raLog := flag.String("ra-log", "", "Path to a single boulder-ra log file")
 	vaLogs := flag.String("va-logs", "", "List of paths to boulder-va logs, separated by commas")
 	timeTolerance := flag.Duration("time-tolerance", 0, "How much slop to allow when comparing timestamps for ordering")
+	checkFromFlag := flag.String("check-from", "", "Day at which to start checking issuances "+
+		"(inclusive). Formatted like '20060102' Optional. If specified, -check-until is required.")
+	checkUntilFlag := flag.String("check-until", "", "Day at which to stop checking issuances "+
+		"(exclusive). Formatted like '20060102'. Optional. If specified, -check-from is required.")
+	flag.BoolVar(&debug, "debug", false, "Enable debug logging")
```

Comment:
Since you treat this as a global anyhow, it's nicer to just declare it directly at the top level:

```
var debug = flag.Bool("debug", false, "Enable debug logging")
```